.travis.yml

```diff
@@ -258,7 +258,7 @@ matrix:
         - ./build-support/bin/travis-ci.sh -ez
 
     # Rust Clippy on Linux with nightly Rust
-    - name: &clippy_shard "[ALLOWED TO BE RED] Rust Clippy on Linux with nightly Rust"
+    - name: "Rust Clippy on Linux with nightly Rust"
```

Comment:
Still "nightly Rust"?

Would it be worthwhile to leave this shard as "allowed_failure", and land the clippy changes as a second commit that unbreaks and locks in clippy? Also fine landing here... up to you. But it's hard to differentiate the formatting changes from the clippy changes.